src/account/AccountContext.tsx

```diff
@@ -18,24 +21,24 @@ const INIT_STATE = {
 
 type StateType = {
   level: string,
-  setActiveAccount: (account: string) => void
-  activeAccount: string | null
-  accounts: { name: string | undefined; address: string }[]
-  setAccounts: (accounts: { name: string | undefined; address: string }[]) => void
+  setActiveAccount: (account: accountType) => void
+  activeAccount: accountType
```

Comment:
`accountType` is an Array.
I didn't understand why React doesn't support an object, this should just work 🤔
As we want to store just the accountId, why not just continue using the `string` type?

Reply:
Changed to save as an object. Don't know why it didn't work before